File: Argentina/Argentina.java
import java.util.Scanner;
public class Program
{
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int ps = in.nextInt();
        int dl = in.nextInt();
        if(ps * 0.02 < dl)
        System.out.println("Pesos");
        else
        System.out.println("Dollars");
    }
}

File: Argentina/Argentina.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int ps,dl;
ps = Convert.ToInt32(Console.ReadLine());
dl = Convert.ToInt32(Console.ReadLine());
if(ps * 0.02 < dl)
Console.WriteLine("Pesos");
else
Console.WriteLine("Dollars");
        }
    }
}
